### PR TITLE
Jelatobranch merge

### PR DESCRIPTION
**Pull request by digitalbrilliancehour**

### CODE REVIEW

**Comment by digitalbrilliancehour** [comment]
I'm having an issue with the sounds, please fix

File: build/Logs/OpenBorLog.txt [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+Game Selected: ./Paks/My Mod.pak
+
+FileCaching System Init......	Disabled
+Initializing video............
+Reading video settings from 'data/video.txt'.
+
+Colordepth is depreciated. All modules are displayed with a 32bit color screen.
+
+Initialized video.............	320x240 (Mode: 0)
```
Comment (digitalbrilliancehour) [inline build/Logs/OpenBorLog.txt:11]: @MikeJones2x this code sucks.

Comment (MikeJones2x) [reply build/Logs/OpenBorLog.txt:11]: Wow

Comment (MikeJones2x) [reply build/Logs/OpenBorLog.txt:11]: @digitalbrilliancehour Wow :(

**Pull request closed without merging**